mne/viz/utils.py

```diff
@@ -3060,6 +3060,14 @@ def _convert_psds(psds, dB, estimate, scaling, unit, ch_names):
     return ylabel
 
 
+def _check_psd_fmax(inst, fmax):
+    """Make sure requested fmax does not exceed Nyquist frequency."""
+    if np.isfinite(fmax) and (fmax > inst.info['sfreq'] / 2):
+        raise ValueError('Requested fmax ({} Hz) must be not exceed 2× the '
+                         'sampling frequency of the data ({}).'
+                         .format(fmax, 2 * inst.info['sfreq']))
```

Comment:
it should be 0.5 sfreq. 100Hz sampling has a max freq of 50Hz

Reply:
@agramfort thanks for catching that. Fixed (also rebased and force-pushed to get travis working); ready for review.